### PR TITLE
Add tests for AddAppointment

### PR DESCRIPTION
**Pull request by tohpinren**
Add tests for AddAppointmentCommand

### CODE REVIEW

**Review by wj331 — DISMISSED** [review]
LGTM. Nice tests added!

File: src/test/java/seedu/address/logic/commands/AddAppointmentCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+    }
+
+    @Test
+    public void execute_appointmentAcceptedByModel_addSuccessful() {
+        ModelStubAcceptingAppointmentAdded modelStub = new ModelStubAcceptingAppointmentAdded();
+        AddAppointmentCommand addAppointmentCommand = new AddAppointmentCommand(INDEX_FIRST_PERSON,
+                "Health check up", LocalDateTime.of(2024, 1, 20, 12, 0));
+        try {
+            CommandResult commandResult = addAppointmentCommand.execute(modelStub);
+            Appointment newAppointment = new Appointment("Health check up",
```
Comment (simbayippy) [inline src/test/java/seedu/address/logic/commands/AddAppointmentCommandTest.java:67]: could you change these test cases to now include bryans addition of Doctor as a required field

**Review by simbayippy — CHANGES_REQUESTED** [review]
needs update as Appointment now includes compulsory Doctor field

File: src/test/java/seedu/address/logic/commands/AddAppointmentCommandTest.java (hunk 1 above)
Comment (tohpinren) [reply src/test/java/seedu/address/logic/commands/AddAppointmentCommandTest.java:67]: @simbayippy Updated please check

**Review by simbayippy — APPROVED** [review]

**Pull request merged**